### DISCUSSION TITLE
It is possible to write operationId to meta with pluginTanstackQuery?

### DISCUSSION THREAD

**Discussion by RabbitShare** [kubb-labs/kubb#1278] [General]
I need add operationId into meta in react-query. Is kubb has some ways to do this?
Or maybe there is a way to find out the operationId when executing a request?

**Comment by stijnvanhulle** [comment]
@RabbitShare Could this help: https://github.com/kubb-labs/kubb/blob/42d661299f3f7f03d1415fd8cc67cc7089774c34/examples/client/src/gen/clients/axios/operations.ts#L4. A operations.ts file is created that contains as key operationId and value the path and method.
